src/Calendar/Calendar.js

```diff
@@ -740,6 +756,8 @@ Calendar.propTypes = {
     }),
     /** Additional props to be spread to the month\'s `<table>` element */
     monthListProps: PropTypes.object,
+    /** Set to **true** if the Today button should be displayed */
+    showToday: PropTypes.object,
```

Comment:
Trying to do this using a localizable `todayLabel`: https://github.com/SAP/fundamental-react/pull/964

Reply:
Ah yeah I forgot to localize the label, thanks